### DISCUSSION TITLE
Difference between "Synology_enable_M2_volume" and " Synology_HDD_db"

### DISCUSSION THREAD

**Discussion by szerwony** [007revad/Synology_enable_M2_volume#198] [Q&A] [answered]
Hi. I have DS1522+ with DSM 7.2.2 and I would like to make a storage pool on my M.2 Lexar SSD. Hovewer I don't understand the difference between "Synology_enable_M2_volume" and " Synology_HDD_db". It looks like " Synology_HDD_db" does more than "Synology_enable_M2_volume" as it allows more RAID types and also supports external PCie cards. So why would anyone want to use "Synology_enable_M2_volume"? I can't understand what is the purpose of it and why would I want to choose one over another.

**Comment by 007revad** [comment, marked as answer]
Originally Synology_HDD_db just added your drives to DSM's compatible drive database and Synology_enable_M2_volume made DSM think your M.2 drives are Synology drives. But Synology_HDD_db has grown to do a lot more making Synology_enable_M2_volume redundant IMHO, but many people still use it.

I have a DS1821+ with a E10M20-T1 and 4 WD NVMe drives and I only use Synology_HDD_db.

**Reply by szerwony** [reply]
Thank you, now I see the difference. I will go with Synology_HDD_db

**Reply by szerwony** [reply]
I also wonder what will happen if I won't run Synology_HDD_db again after DSM update. Will my data stored on the SSD be lost?

**Reply by 007revad** [reply]
Your data on the M.2 SSD will be safe, but the drive will show a warning in storage manager.

**Reply by hagagit** [reply]
I had to use the enable script, because the db scripz allone was not enought ( DS423+ / WD_Black SN850X NVMe)
